aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -13,6 +13,21 @@
 
 namespace at::native::mps {
 
+/**
+ * Computes number of elements one needs to transfer to preserve all the elements
+ */
+size_t compute_strided_size(const at::Tensor& t) {
```

Comment:
This computes memory address offsets, that should be clear in the name otherwise it's very confusing with the current definition of "size" in the lib.

Reply:
Would `compute_strided_range` be a more appropriate name here?